### PR TITLE
[DNM] Fix ictest upgrade for PR 597

### PR DESCRIPTION
**Pull request by Reecepbcups**
testing little things to try and fix, DO NOT MERGE - removing other CI for faster times here

Once this works, Ill manually pull in changes to the other PR & close this + delete branch

### CODE REVIEW

File: tests/interchaintest/setup.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ var (
 		Bech32Prefix:        "juno",
 		Denom:               "ujuno",
 		CoinType:            "118",
-		GasPrices:           "0.003ujuno",
+		GasPrices:           "0.0ujuno",
```
Comment (faddat) [inline tests/interchaintest/setup.go]: ujunox?

Comment (Reecepbcups) [reply tests/interchaintest/setup.go]: for ictest we use standard ujuno, not x

Comment (Reecepbcups) [reply tests/interchaintest/setup.go]: per `Denom:               "ujuno",` for the base denom

**Comment by Reecepbcups** [comment]
WOOOOOOOOOOOO Hacky solutions works - will upstream a fix later. Has to be an easier way

**Pull request closed without merging**